truss/templates/trtllm/model/model.py

```diff
@@ -122,8 +123,14 @@ async def generate():
 
         if model_input.stream:
```

Comment:
@pankajroark do you know if this would keep unaddressed the issue we run into with benchmarks, since I believe those invoke inference with streaming=True to calculate TTFT?